website/api/blockchain-integration/eth-examples.md

```diff
@@ -18,8 +18,8 @@ The [RISC Zero Foundry Template][foundry-template] provides a minimal applicatio
 
 This [example app][governance-example] uses Bonsai as an Ethereum coprocessor. The protocol, based on the OpenZeppelin [Governor smart contract standard], batches signature verifications off-chain for a DAO governance vote. The end result is that in [~160 lines of Rust][signature-aggregation], a gas savings of 66% is achieved with significant room for optimizations.
 
-[zkvm-examples]: ../zkvm/examples.md
-[remote-proving]: ../generating-proofs/remote-proving.md
+[zkvm-examples]: /api/zkvm/examples
```

Comment:
For docs in `/website/api` that are pointing to other docs in `website/api`, we should use relative links, for two reasons: 

1. The contents in `website/api/` get rendered under `next` on the website, and links on those pages should point to other docs on `next`. 
2. When we push a new release and move the docs into `website/api_versioned_docs/`, the relative paths will still work.

Reply:
does that mean that _all_ absolute links are wrong? Because I see quite a lot of them atm

Reply:
what I mean is should I change all occurences of `[zkvm-examples]: /api/zkvm/examples` to `[zkvm-examples]: ../zkvm/examples`

Reply:
The only place we need to be using absolute links is for links that go back and forth between docs in the versioning system (i.e. in `website/api/` or in `website/api_versioned_docs/`) and docs that aren't in the versioning system (i.e. those in `website/docs/`, including the terminology page & faq). 

I'm not super concerned about getting rid of all the absolute links from `website/api_versioned_docs/`, but it would be nice from a maintenance perspective for those links to point to the associated content in that version's docs.

Does that answer your question?

Reply:
yeah it does, thanks for that!

OK, well I have another big PR that includes remark linting [here](https://github.com/risc0/risc0/pull/2096), so maybe I add a rule to enforce relative links somehow.. will do some research